solidity/scripts/lcl-initialize.js

```diff
@@ -16,10 +16,32 @@ module.exports = async function () {
     // 50 ETH / 3 members = 16,67 ETH of unbonded value for each member.
     // Here we set the bonding value to bigger value so members can handle
     // multiple keeps.
-    const bondingValue = web3.utils.toWei("50", "ether")
+    let bonding = "50"
+
+    const networkID = await web3.eth.net.getId()
+    // NetworkID 44787 is Celo Alfajores testnet. Bonding is lower becuase
+    // Alfajores faucet funds 10cUSD and 5 CELO at a time.
+    if (networkID == 44787) {
+      bonding = "2"
+    }
+    const bondingValue = web3.utils.toWei(bonding, "ether")
 
     const accounts = await web3.eth.getAccounts()
-    const operators = [accounts[1], accounts[2], accounts[3], accounts[4]]
+    let numberOfAccounts = 1
+    let iter = 0
+    if (accounts.length > 1 && accounts.length <= 5) {
+      numberOfAccounts = accounts.length
+      iter = 1
+    } else if (accounts.length > 5) {
+      numberOfAccounts = 5
+      iter = 1
+    }
+
+    const operators = []
+    for (let i = iter; i < numberOfAccounts; i++) {
+      operators.push(accounts[i])
+    }
```

Comment:
I do not understand this change. The assumption in `solidity/integration/smoke_test.js` is that accounts [1,4] are operators, account 5 is a keep owner, and account 6 is the application. This will work for the second case where we have at least 6 accounts but the first case will fail to complete the smoke test. I also do not think there is a chance this script will work with Alfajores connection, as defined in `truffle.js` - `web3.eth.getAccounts()` will always return only one account (`process.env.CONTRACT_OWNER_CELO_ACCOUNT_PRIVATE_KEY)`.

Instead of adding these conditions and modifying bonding value earlier, I'd recommend exiting this script if the funding account does not have enough funds or if there are no at least 6 accounts in the system.

At the same time, we should update celo Docker image introduced in https://github.com/keep-network/keep-core/pull/2324 to create one more account.

With this logic, we'll be able to work with Celo locally the same way as we do with Ethereum. Alfajores will not be fully initialized but the same issue is for Ropsten for which we use separate provisioning scripts.

Reply:
Right, `smoke_test.js` won't work neither on Alfajores nor Ropsten with the current config. But isn't `smoke_test.js` is just one of our use cases? I mean, if you'd want to test some other functionality on testnet without having all 6 accounts at our disposal then running this script might still be helpful? This script authorizes, deposits and creates a new sortition pool for an application. With the current setup `initialize-celo.sh` will pass and replace all the addresses needed to run the client. 

On the other hand I see your point that if we don't have a full support for testnets yet, then anything between might just be confusing. If we'd rather not deal with testnets for now in these scripts, then I can certainly revert changes in `lcl-initialze.js` but also will need to modify `initialize-celo.sh` to inform that we can initialize only local network.

> At the same time, we should update celo Docker image introduced in keep-network/keep-core#2324 to create one more account

Sure, I will create a new account.

Reply:
To open a new keep on any network, local or testnet, we need at least 3 accounts for operators. On Alfajores, we'll always have one when running this script. Agreed, sortition pool initialization is something we can do but as you say - let's either fully execute the initialization or not execute it at all and leave it for separate scripts, just like we do for Ropsten.
I like your idea about executing this script only for `local`. This is cleaner than exiting from it.

Reply:
221eba0 & 2ccc305